taipy/gui/viselements.json

```diff
@@ -1512,6 +1512,11 @@
                         "type": "dynamic(Union[str,list[str]])",
                         "doc": "Semicolon (';')-separated list or a list of menu items identifiers that are disabled."
                     },
+                    {
+                        "name": "selected_items",
```

Comment:
I would prefer "selected_ids" for short and homogeneity.
By the way, the properties order is twisted. I would recommend:
'lov', 'label', 'on_action', 'selected_ids', 'inactive_ids', 'adapter', 'type', 'width', 'width[mobile]'.

Then the question is: how do we select/unselect items. Users have to handle that in `on_action` I imagine?

Reply:
I've added a logic to handle the select/unselect items. User no need to use on_action anymore

Reply:
I guess you're saying that's done in 'on _change' which is fine.
However 'on_action' is still invoked right (independent on the selected state of the item)?